spade/core/project.py

```diff
@@ -1,55 +1,17 @@
 import datetime
-from sqlalchemy import create_engine, MetaData, Table, Column, Binary, Integer, String, ForeignKey
-
-class Project:
-    """Represents an open session for spade."""
 
-    def __init_db(self, engine):
-        """
-        Creates default tables for a newly created spade project
-        """
-        metadata = MetaData()
-
-        # Define project_info table
-        self.table_pinfo = Table("project_info", metadata,
-            Column("key", String, primary_key=True),
-            Column("value", String)
-        )
+from sqlalchemy import create_engine, MetaData, Table, Column, Binary, Integer, String, ForeignKey
 
-        # Define files table
-        self.table_files = Table("files", metadata,
-            Column("id", Integer, primary_key=True, autoincrement=True),
-            Column("path", String, unique=True),                # Relative or absolute path to this file
-            Column("hash", Binary),                             # sha256 hash of file contents on last change
-            Column("head_change", Binary, server_default=None), # head change file is at
-        )
+SCHEMA_VERSION = "0.0a"
```

Comment:
You define this var but then never use it (it should be used in constructor where we set schema version project info variable).  Also I don't want to use a strict versioning system with major and minor changes for this, lets just use a nonce and up the number each time we change the schema.